FinalTestRewriteTask1/stringLexer.c

```diff
@@ -0,0 +1,74 @@
+#include <stdbool.h>
+#include <string.h>
+#include <stdio.h>
+#include "stringLexer.h"
+
+enum states{startState, firstClosureCharacter, readAtCharacter, secondClosureCharacter, readDot ,errorSate};
```

Comment:
```suggestion
enum States {
    startState, 
    firstClosureCharacter, 
    readAtCharacter, 
    secondClosureCharacter, 
    readDot,
    errorState
};
```
Какой-то ужас со стайлгайдом